RevenueCatUI/Data/TestData.swift

```diff
@@ -103,6 +103,18 @@ enum TestData {
         introductoryDiscount: Self.intro(14, .day, priceString: "$1.99"),
         locale: Self.locale
     )
+    static let annualProduct60 = TestStoreProduct(
+        localizedTitle: "Annual",
+        price: 59.99,
+        localizedPriceString: "$59.99",
+        productIdentifier: "com.revenuecat.product_3",
+        productType: .autoRenewableSubscription,
+        localizedDescription: "PRO annual",
+        subscriptionGroupIdentifier: "group",
+        subscriptionPeriod: .init(value: 1, unit: .year),
+        introductoryDiscount: Self.intro(14, .day, priceString: "$2.99"),
+        locale: Self.locale
+    )
```

Comment:
I added this because calculated prices (eg "price per month" for a product that is priced in cost per year, so the price per month is calculated as (cost per year)/12) are *not* rounded *except* if they end in .99 or .00. So 59.99/12 = 4.999 is used for some tests.